test/PSR7SessionTest/Http/SessionMiddlewareTest.php

```diff
@@ -284,7 +297,7 @@ public function testAllowsModifyingCookieDetails()
         self::assertSame($defaultCookie->getPath(), $tokenCookie->getPath());
         self::assertSame($defaultCookie->getHttpOnly(), $tokenCookie->getHttpOnly());
         self::assertSame($defaultCookie->getMaxAge(), $tokenCookie->getMaxAge());
-        self::assertEquals(time() + 123456, $tokenCookie->getExpires(), '', 2);
+        self::assertEquals($clock->now()->getTimestamp() + 123456, $tokenCookie->getExpires());
```

Comment:
Assertion should not reuse the `$clock`, but just the fake value passed to the `FakeClock` constructor (reducing indirection)